src/core_plugins/kibana/public/dashboard/grid/_dashboard_grid.scss

```diff
@@ -57,10 +57,6 @@
 
 // REACT-GRID
 
-.react-grid-layout .dshPanel {
-  overflow: visible;
```

Comment:
This can safely be removed. It was needed when InputControls use `Select` and the context menu was `KuiContextMenu`. Now Eui is used for all of these and Eui uses portal to attach overlay elements to body so there are no more clipping issues